### PR TITLE
conda-forge install instructions

### PR DESCRIPTION
**Pull request by scopatz**
This is a small edit to tell conda users how to install the themes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,12 +54,14 @@ the previews for each theme in the [section](#previews) below or in this other
...
     include ./theme.conf
     ```
-    
+
+### Conda
+
 If you using the ``conda`` package manager, you may also install these themes
 with the following command:
 
     ```bash
```
Comment (dexpota) [inline README.md]: This block of code is not being rendered correctly. I think the problem is with the indentation, try to remove it.

Screenshot 2019-06-29 09:56:08

**Comment by scopatz** [comment]
Ok, fixed! Sorry this took me so long to get back to.

**Comment by dexpota** [comment]
@all-contributors please add scopatz for 📖

**Comment by dexpota** [comment]
@all-contributors please add @scopatz for doc

**Pull request merged**